docs/faq.rst

```diff
@@ -297,23 +297,18 @@ directory specified prior to a ``launch --local-entry`` command
 
 Why should I migrate my existing workflow to use GitHub Apps and a .dockstore.yml?
 ----------------------------------------------------------------------------------
-Version 1.9.0 has added support for fetching workflows using GitHub Apps. Installing our GitHub App
+Release 1.9.0 of Dockstore has added support for syncing workflows using GitHub Apps. Installing our GitHub App
```

Comment:
Should also say something about it being more efficient from a user perspective.
i.e. we only pull the versions that have changed, so it is faster